### PR TITLE
WP-14889 Import failure from Snowflake

### PR DESCRIPTION
**Pull request by sunil-varicent**
Ticket: https://varicent.atlassian.net/browse/WP-14889

### CODE REVIEW

File: tap_snowflake/__init__.py [hunk 1]
```diff
@@ -321,6 +328,11 @@ def resolve_catalog(discovered_catalog, streams_to_sync):
         # These are the columns we need to select
         columns = desired_columns(selected, discovered_table.schema)
 
+        if (len(columns) > MAX_COLS):
```
Comment (sunil-varicent) [inline tap_snowflake/__init__.py:331]: this is only change, other are auto format changes

Comment (mjdoor) [reply tap_snowflake/__init__.py:331]: Maybe different wording here, cannot vs could not? `Number of columns cannot exceed {MAX_COLS}`

File: tap_snowflake/__init__.py [hunk 2]
```diff
@@ -321,6 +328,11 @@ def resolve_catalog(discovered_catalog, streams_to_sync):
...
+            raise Exception(f'Number of columns cannot exceed {MAX_COLS} ')
```
Comment (mjdoor) [inline tap_snowflake/__init__.py:332]: Can we get rid of the trailing space too?

**Review by mjdoor — APPROVED** [review]

File: tap_snowflake/__init__.py (hunk 2 above)
Comment (sunil-varicent) [reply tap_snowflake/__init__.py]: done

**Review by jayma91 — APPROVED** [review]

**Pull request merged**